package.json

```diff
@@ -3,14 +3,15 @@
   "version": "0.1.0",
   "description": "OpenAI Authentication Library for ChatGPT",
   "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "author": "Allan Oricil <allan@oricil.com>",
   "license": "MIT",
   "engines": {
     "node": ">=14"
   },
   "scripts": {
     "build": "babel src/index.js -d dist",
-    "prepare": "husky install",
+    "prepare": "husky install && cp src/index.d.ts dist",
```

Comment:
this file can be copied over to dist using rollup.
https://www.npmjs.com/package/rollup-plugin-copy